azurerm/resource_arm_cosmosdb_mongo_collection.go

```diff
@@ -66,6 +66,13 @@ func resourceArmCosmosDbMongoCollection() *schema.Resource {
 				ValidateFunc: validation.IntAtLeast(-1),
 			},
 
+			"throughput": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Default:      400,
+				ValidateFunc: validation.IntAtLeast(400),
```

Comment:
We should validate that it is increments of 100 & is at least 400

Reply:
I added a validator that checks for a minimum of 400 and that its set in increments of 100 - I believe this logic applies to other CosmosDB resources as well, so I added it to the existing `cosmos` validators.

I also saw that there were no test cases for the existing cosmos validators, so I added them in